.gitignore

```diff
@@ -3,4 +3,4 @@ npm-debug.log
 build
 generated
 .antwar
-
+.vscode
```

Comment:
I've not yet reviewed this fully, but one other thing: there seem to be two conflicting files. Merge or rebase to fix it.